File: java/Product.java
public class Product {
    
    // deklarasi dan inisialisasi atribut private
    private int price = 0;
    private String idProduct = "-";

    
    // constructor tanpa parameter
    public Product(){

    }

    // constructor dengan parameter
    public Product(int price, String id){
        this.price = price;
        this.idProduct = id;
    }


    // deklarasi setter sebagai method (Write)
    public void setPrice(int price){
        this.price = price;
    }
    public void setIdProduct(String id){
        this.idProduct = id;
    }

    // deklarasi getter sebagai method (Read Only)
    public int getPrice(){
        return this.price;
    }
    public String getIdProduct(){
        return this.idProduct;
    }

    // deklarasi method Display untuk menampilkan data menggunakan method Getter
    public void displayProduct(){
        System.out.println("Price        : $" + this.getPrice());
        System.out.println("ID Product   : " + this.getIdProduct());
    }
}

File: python/Product.py
class Product():
    
    # deklarasi dan inisialisasi atribut private
    __price = 0
    __idProduct = 0

    
    # constructor dengan parameter
    def __init__(self, price = 0, idProduct = 0):
        self.__price = price
        self.__idProduct = idProduct

    
    # deklarasi setter sebagai method (Write)
    def setPrice(self, price):
        self.__price = price

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    
    # deklarasi getter sebagai method (Read Only)
    def getPrice(self):
        return self.__price

    def getIdProduct(self):
        return self.__idProduct


    # deklarasi method Display untuk menampilkan data menggunakan method Getter
    def displayProduct(self):
        
        print("Price        : $" + str(self.getPrice()))
        print("ID Product   : " + str(self.getIdProduct()))